recipe-generator-node/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "recipe-generator",
+    "version": "0.3.4",
+    "description": "Node wrapper to run generation for the Recipe Framework",
+    "author": "Derek Cormier",
+    "homepage": "https://github.com/kormide/recipe",
+    "license": "MIT",
+    "keywords": [
+        "recipe",
+        "ingredient",
+        "integration",
+        "testing"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/kormide/recipe"
+    },
+    "private": false,
+    "main": "index.js",
+    "files": [
+        "index.js"
+    ],
+    "bin": "./index.js",
+    "devDependencies": {
+        "eslint": "5.9.0"
+    },
+    "dependencies": {
+        "argparse": "1.0.10"
```

Comment:
Does this version need to be so precise? When you're writing a util library like this it's nice to allow other versions to avoid the user having to have multiple versions of `argparse`. The norm would be `^1.0.10`, assuming `argparse` following semantic versioning properly.